Sources/EpoxyLayoutGroups/Groups/HGroup.swift

```diff
@@ -22,6 +22,7 @@ public final class HGroup: UILayoutGuide, Constrainable, InternalGroup {
     alignment = style.alignment
     accessibilityAlignment = style.accessibilityAlignment
     spacing = style.spacing
+    reflowsForAccessibilityTypeSizes = style.reflowsForAccessibilityTypeSizes
```

Comment:
Now that we take this in from the style in the initializer can we remove the default value of `true` on line 166 since it will always be overridden?